spec/components/finance/statements/payment_breakdown/ecf_spec.rb

```diff
@@ -86,29 +87,14 @@
     expect(counts_list[2][0]).to eq("Total completed")
     expect(counts_list[2][1]).to eq("30")
 
-    expect(counts_list[3][0]).to eq("Total voided")
-    expect(counts_list[3][1]).to eq("50")
-  end
-
-  describe "with extended count" do
```

Comment:
could we keep the test or amend this one to include the extended count being more than 0?